src/pages/[platform]/build-a-backend/storage/download/index.mdx

```diff
@@ -149,16 +170,35 @@ const { body, eTag } = await downloadData({
   }
 }).result;
 ```
+</Block>
+</BlockSwitcher>
 
 ### Get the text value of downloaded File
 
 You can consume the value of file in any of the three formats: `blob`, `json`, or `text`. You can call the respective method on the `body` property to consume the set data in the respective format.
 
+<BlockSwitcher>
+<Block name="With Path (Above v6.x.x)">
+```javascript
+import { downloadData } from 'aws-amplify/storage';
+
+try {
+  const downloadResult = await downloadData({ path: "/album/2024/1.jpg" }).result;
```

Comment:
nit: let's wrap the long line to avoid possible horizontal scrolling

Reply:
nice nit. didn't realize this one